### ISSUE TITLE
Abort XHR in progress

### ISSUE THREAD

**Issue by dotBits** [marmelab/restful.js#84]
Wouldn't be useful a method that aborts a specific xhr request?

**Comment by David-Hari** [comment]
I'd like that too.

**Comment by RobinBressan** [comment]
Hi, by design it is not possible to cancel a HTTP request. It should be possible to fake this behavior by rejecting the request promise but it will be faked (the request will be performed anyway).

**Comment by David-Hari** [comment]
It's certainly possible to [cancel an XMLHttpRequest](https://www.google.com.au/search?q=cancel+xhr).
I know this library uses the new request stuff, but I'd hope it were possible there too.

**Comment by dotBits** [comment]
@David-Hari yes, [the abort method](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/abort) should be used
